core/templates/domain/feedback_message/ThreadMessage.model.ts

```diff
@@ -26,8 +26,12 @@ export interface ThreadMessageBackendDict {
   'entity_id': string;
   'message_id': number;
   'text': string;
-  'updated_status': string;
-  'updated_subject': string;
+  // These properties can be null as we have defined functions
+  // 'hasStatusUpdate' & 'hasSubjectUpdate' to check for them to be not null,
+  // the corresponding tests with null values for these properties can
+  // be seen in FeedbackThreadObjectFactorySpec.ts file.
```

Comment:
Just because a value can null doesn't mean it has to be. A better explanation would be why a null value makes sense for this variable. This could be a little confusing at first. Let me know if you are confused with how the comment would look like.

Reply:
Updated, please take a look!